### DISCUSSION TITLE
Modernize branding (name & icon)

### DISCUSSION THREAD

**Discussion by alizehctl** [CodedOre/NewCaw#33] [General]
Cawbird is probably my favourite Twitter app, and I'm really looking forward to seeing it modernized with GTK4 & libadwaita. The only thing that has held me back from using it very often is its branding - both the icon and name.

It would be great if along with this rewrite a new name and icon could be minted for the app.

I don't have the skills to craft a new icon for it, but I'd be happy to help brainstorm new names for it. Ideally the name would conform to GNOME's [App Naming Guidelines](https://developer.gnome.org/hig/guidelines/app-naming.html).

**Comment by alizehctl** [comment]
Names of common bird species are a natural starting point:
- Nightingale
- Canary
- Pelican
- Hummingbird
- Raven

**Reply by CodedOre** [reply]
If we are discussing to change the name, I think we should rather go from what the application does, rather than bird names.

At the end, we are likely to include Mastodon support as well, so it's not just related to the bird platform Twitter any more.

**Reply by IBBoard** [reply]
I think we're slightly better on branding than the old Corebird with its weird "blue bird with pink brain on show" icon 😄 I think the key question is how you sum up "short-form social media" in a single term. And the bit that @CodedOre said while I'm typing this - good luck getting a bird name that also fits with Mastodon!

I will say that I *hate* the current trend of Gnome icons, though. The art is fine in general (give or take some over-saturation) but the [Tango Project](https://web.archive.org/web/20060405032038/http://tango-project.org:80/Tango_Icon_Theme_Guidelines) had it right with *clear and obvious outlines*. Too many recent Gnome icons are turning into boxes.

**Reply by alizehctl** [reply]
That's a good point.

If we are going to base the identity of the app on its function instead of the brand of the social network, then we can come up with a name that has something to do with microblogging. It's an abstract concept though and I'm struggling to think of a catchy associated name.

**Reply by alizehctl** [reply]
Some new name ideas considering the app will be multi-platform:
- Shortform
- Micro-post
- Threaded
- Shout
- Broadcast

**Reply by IBBoard** [reply]
Shortform and Micro-Post fit the micro-blogging idea.

Threaded could work, but there's probably a lot of interpretations. And how often do people thread conversations rather than just post lots of one-off messages?

I can see where you're going with Shout and Broadcast, but they're probably better for streaming apps 🙂

Is "Status" too short and generic? There was Status.net, which is now Identi.ca. And "status update" redirects to the microblogging page on Wikipedia.

**Reply by IBBoard** [reply]
Mastodon is based on [ActivityPub](https://en.wikipedia.org/wiki/ActivityPub). But "Activity" probably makes people think of activity trackers or the [Gnome Activity Journal](https://en.wikipedia.org/wiki/GNOME_Activity_Journal).

We could go Micro-Blogger or Micro-Social.

I don't know how any of these could be turned into icons, though!

**Reply by alizehctl** [reply]
I do feel status is a bit too generic, since you can post statuses on Facebook too. I think Microblogger could be a really good descriptive name. My only concern is that it may be misconstrued as a client for micro.blog...

but I don't think it's a huge concern as the concept of microblogging is much more popular than the micro.blog network

**Reply by IBBoard** [reply]
I wasn't even aware that micro.blog existed! It's definitely in the realms of having web.server and email.client and complaining that people make their own "web server" and "email client" apps.

As for Status, you can post a status to other services like Facebook, but we don't have to support all status locations and my web host [posts statuses about the servers](https://status.mythic-beasts.net.uk/) so "Status" no tool is going to cover all forms of "status".

**Reply by alizehctl** [reply]
Thinking about it again after a while - I think Microblogger is the best name out of the options listed so far.

**Comment by CodedOre** [comment]
I also already got the mention from the GNOME Design Team that the icon could need an update.
So, I was already thinking about an updated icon.

However, I think this should be rather discussed with @IBBoard, especially the thing with changing the name.

Other than that, should we agree on updating these I think we can ask the Design Team for help as well.

**Comment by CodedOre** [comment]
@IBBoard, since you mentioned that we need to consider a renaming in IBBoard/cawbird#452, does this implies that you would want to change the name for Cawbird 2.0?

Not really sure how I should interpret your sentence there.

**Reply by IBBoard** [reply]
If we're moving closer to the Gnome circle with Adwaita and appearing in "This Week in Gnome" then I think it's worth considering a new name that fits better with the naming guidelines. As there's little/no common code then it's effectively only Cawbird by spiritual lineage rather than code heritage.

It might be that we can't come up with anything better. But it might be that we can find a more suitable cross-platform name, at which point the new icon is great but won't get as much use 🙂

**Reply by IBBoard** [reply]
I don't know if it's worth revisiting this.

Since Twitter banned third-party clients, NewCaw is now only going to support Mastodon. We _could_ rename to something more Mastodon-y. But it's not mandatory, and all of the best names have gone.

Or we could just keep with Cawbird for familiarity, even though it's not themed for the platform.

**Reply by alizehctl** [reply]
There are already a couple of mastodon themed linux clients:
- Tootle
- Tooth
- Tokodon
- Toot

I think it'd be nice if the app name was less specific to Mastodon and more about microblogging on the fediverse.
Some potential names in that theme:
 - Microverse (perhaps too similar to metaverse)
 - Microblogger
 - Shortform
 - Micro-post
 - Fedipost
 - miniblog
 - Fedistream
 
I noticed some clients use the names of brass wind instruments (after the "toot" sound). Some names in that theme:
- Trumpet
- Trombone
- Euphonium

Some more mastodon-y names:
- Tooter
- Tootr
- Pachyderm
